src/Fork.php

```diff
@@ -64,16 +64,9 @@ public function call(callable $func, $args = null)
                     call_user_func_array($func, $args);
                 }
             } catch(\Exception $e) {
-                $memory = shmop_open($this->memoryKey, "c", 0644, static::SHARED_MEMORY_LIMIT);
-                $errors = shmop_read($memory, 0, static::SHARED_MEMORY_LIMIT);
-                $errors = trim($errors);
-                if ($errors) {
-                    $errors .= "\n";
-                }
-                $errors .= "Exception: " . $e->getMessage() . " (" . $e->getFile() . ":" . $e->getLine() . ")";
-                shmop_write($memory, $errors, 0);
-                shmop_close($memory);
-                exit(1);
+                $this->onException($e);
```

Comment:
Thanks @bkrukowski, we'll be dropping support for `5.6` soon, so this pr can just target `7.0` and only worry about `\Throwable`, we don't need a separate catch for `\Exception`